app/src/main/res/values-ar-rSA/strings.xml

```diff
@@ -184,6 +184,7 @@
     <string name="preference_manage_groups_summary">أدر واحذف مجموعاتك هنا</string>
```

Comment:
Please remove the translations to other languages. All translations are managed in Crowdin and the ones you've added here would be overwritten when we import them before the next release.